### ISSUE TITLE
No suggestion for map-style constructor invocation by code assist when the class being constructed is written in Java

### ISSUE THREAD

**Issue by mauromol** [groovy/groovy-eclipse#332]
This is the old GRECLIPSE-1792.

Consider the following:

```java
package c;

public class MyBeanJava {
	private String foo;
	
	public String getFoo() {
		return foo;
	}
		
	public void setFoo(String foo) {
		this.foo = foo;
	}
	
	public void setBar(String bar) {
		foo = bar.toUpperCase();
	}
}
```

And:

```
package c

class ConstructorTest2 {
	static main(args) {
		MyBeanJava b = new MyBeanJava(|)
	}
}
```

Invoke code assist at "|": no suggestion is given to perform a map-style constructor invocation.

Please note this code is the same as #330, with the only difference being the "bean" class is written in Java instead of Groovy.

**Comment by eric-milles** [comment]
Could you give this a re-try with the latest build?

**Comment by mauromol** [comment]
This use case seems to be fixed in 2.9.2.xx-201709052200-e46, thank you! 👍
